CourseManager/ViewModels/CourseDetailsViewModel.swift

```diff
@@ -16,6 +16,7 @@ class CourseDetailsViewModel: ObservableObject {
     
     func goToSecondaryApp() {
         //hardcoded values to be replaced soon
+        //update CourseInformation model to include appPath
```

Comment:
@sghourab you can use #warning("message"), so it will show warnings in Xcode project, so you can easily find it when you want to refactor

Reply:
done